app/javascript/components/side-panel.jsx

```diff
@@ -26,20 +28,33 @@ const FixedContent = styled.div`
   color: ${colors.white};
 `;
 
+const getSidePanel = gql`
+  fragment SidePanel on Location {
+    ...Wifi
+    ...Bathroom
+  }
+
+  ${Wifi.fragments.location}
+  ${Bathroom.fragments.location}
```

Comment:
I'm not a huge fan of how we're pulling the fragments in here. The `.fragments.location` feels like it's an implementation detail of the `withFragments` HOC.

Can we explicitly export these from their respective files and pull them in here as named imports? That way it makes it explicit to the reader that these are being referenced somewhere other than the file where they're defined.